storage/index.cpp

```diff
@@ -5,13 +5,18 @@
 
 namespace storage
 {
-  // GCC bug? Can't move initialization to hpp file (linker error).
-  const int TIndex::INVALID = -1;
-
+  const string TIndex::INVALID = string("");
   string DebugPrint(TIndex const & r)
   {
-    ostringstream out;
-    out << "storage::TIndex(" << r.m_group << ", " << r.m_country << ", " << r.m_region << ")";
-    return out.str();
+    return r.m_idx;
```

Comment:
Yes. But now typedef of std::string is used instead of TIndex structure. This code had been deleted.